app/controllers/organisation_relationships_controller.rb

```diff
@@ -14,7 +14,7 @@ class OrganisationRelationshipsController < ApplicationController
   ]
 
   def stock_owners
-    stock_owners = organisation.stock_owners
+    stock_owners = organisation.stock_owners.filter_by_active
```

Comment:
Ooh, I was wondering how we'd _restore_ these upon reactivation, and it looks like this might be all that's needed, nice